docs/docs/deploying-to-gitlab-pages.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Deploying to GitLab Pages
+---
+
+GitLab Pages are similar to GitHub Pages, perhaps even easier to setup. It also
```

Comment:
In the Style Guide, it says to avoid say "easier" (especially to help beginners not worry if it's not easy for them) and be more specific. How are GitLab pages easier than GitHub pages to set up? Does it take fewer steps?